unison-src/builtin-tests/jit-tests.output.md

```diff
@@ -9,4 +9,6 @@ TODO remove md5 alias when base is released
 ```ucm
 .> run.native tests
 
+  Scheme evaluation failed.
+
```

Comment:
Can you check in a version that includes output from the tests? Guessing you got this running locally but maybe just didn't refresh this file.